junit-engine-api/src/main/java/org/junit/gen5/engine/ExecutionRequest.java

```diff
@@ -25,12 +25,16 @@
 
 	private final EngineExecutionListener engineExecutionListener;
 
+	private ConfigurationParameters configurationParameters;
+
 	private final Map<String, Object> attributes = new HashMap<>();
```

Comment:
IIRC `attributes` were added for exactly the purpose this pull request is solving.

BTW please make the instance variable `final`.